### PR TITLE
Added Hindi Translated strings #2365

### PR DESCRIPTION
**Pull request by SekhGulamMainuddin**
Fixes #2365 

Added Hindi Translations for the Strings.

Please add #hacktoberfest on merge.

### CODE REVIEW

**Comment by SekhGulamMainuddin** [comment]
@therajanmaurya please merge my PR.
Please add hacktoberfest tags on merge

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -553,7 +553,7 @@
     <string name="privacy_policy_host" translatable="false">openmf.github.io/privacy_policy_mifos_mobile.html</string>
     <string name="privacy_policy_host_url" translatable="false">https://openmf.github.io/privacy_policy_mifos_mobile.html</string>
     <string name="transfer_error">Unable to complete transfer, please try again later</string>
-    <string name="please_wait">Please Wait...</string>
+    <string name="please_wait">Please Wait…</string>
```
Comment (PratyushSingh07) [inline app/src/main/res/values/strings.xml]: I think the earlier string looked better . The new one looks congested

**Review by PratyushSingh07 — CHANGES_REQUESTED** [review]
Also do these translations cover the entire app ? I meant do we need to add more hindi translations for our mobile applications or are these enough ?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (SekhGulamMainuddin) [reply app/src/main/res/values/strings.xml:556]: The Android Editor recommended it. And its the same

**Comment by SekhGulamMainuddin** [comment]
> Also do these translations cover the entire app ? I meant do we need to add more hindi translations for our mobile applications or are these enough ?

I have translated all the strings to Hindi. And latter on if we add more we need to do the changes simultaneously

**Pull request closed without merging**